content/blog/learn-what-you-can-do-with-hpe-data-services-cloud-console-api-in-just-3-minutes.md

```diff
@@ -0,0 +1,201 @@
+
+Finally click the **Send** button. You will get a JSON representation of the storage system resources registered based on the query parameters specified. Here, I am getting the list of storage systems of type “*HPE Alletra 9060*”.
+
+```json
+{
+    "items": [
+        {
+            "systemWWN": "2FF70002AC07EB29",
+            "name": "s3294",
+            "id": "4UW0003294",
+            "description": "System s3294",
+            "mgmtIp": "16.182.41.5",
+            "softwareVersions": "9.5.0",
+            "model": "HPE Alletra 9060",
+            "productFamily": "deviceType1",
+            "state": "NORMAL",
+            "callhomeStatus": "ENABLED_NORMAL",
+            "resourceUri": "/api/v1/storage-systems/device-type1/4UW0003294",
+            "upSince": 1670726169000,
+            "fqdn": "s3294.mip.storage.hpecorp.net",
+            "capacityDetail": {
+                "volumeSpace": 21092875,
+                "snapSpace": 1676167,
+                "totalUsedSpace": 22769042
+            },
+            "associatedLinks": [
+                {
+                    "type": "storage-pools",
+                    "resourceUri": "/api/v1/storage-systems/4UW0003294/storage-pools"
+                },
+                {
+                    "type": "volumes",
+                    "resourceUri": "/api/v1/storage-systems/4UW0003294/volumes"
+                },
+                {
+                    "type": "swupdatestatus",
+                    "resourceUri": "/api/v1/storage-systems/4UW0003294/swupdate/status"
+                }
+            ],
+            "arrayList": null,
+            "collectionStatus": {
+                "metricStatus": {
+                    "status": "NORMAL"
+                },
+                "configStatus": {
+                    "status": "NORMAL"
+                },
+                "overAllStatus": "NORMAL"
+            },
+            "lastConnectedTime": 1674674119,
+            "connectionStatus": "CONNECTED",
+            "customerId": "c93de31c382811ecb478320c1ce21c93",
+            "generation": 1674674119,
+            "type": "storage-system",
+            "tierType": "STORAGE_TIER_9000_NVME",
+            "tierName": "HPE Alletra 9000"
+        }
+    ],
+    "total": 1,
+    "pageLimit": 50,
+    "pageOffset": 0,
+    "requestUri": "https://us1.data.cloud.hpe.com/api/v1/storage-systems?filter=model%20eq%20%22HPE%20Alletra%209060%22"
+```
+
+**That’s it. No more than 3 minutes!**
+
+## Summary
+
+This blog gives you a great example on how to obtain the access token using API call and help you get started with the Data Services Cloud Console REST API for the *HPE GreenLake for Block Storage* data service using Postman. Additional HPE data services APIs will be published in accordance to the expansion of cloud data services delivered through the HPE Data Services Cloud Console. So make sure to stay tuned for any update of Mark’s public Postman collection for Data Services Cloud Console API.  
```

Comment:
Add the word "an" between "using" and "API call" in the first sentence.  Add the word "should" prior to "help you get started".

3rd sentence: Take off "So" from the beginning and just start with "**Make sure you** stay tuned for any update.." (change "to" to "you" as well)